notifications_api_common/management/commands/register_kanaal.py

```diff
@@ -0,0 +1,79 @@
+import logging
+
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand
+from django.urls import reverse
+
+from ...kanalen import KANAAL_REGISTRY
+from ...models import NotificationsConfig
+
+logger = logging.getLogger(__name__)
+
+
+class KanaalExists(Exception):
+    pass
+
+
+def create_kanaal(api_root: str, kanaal: str) -> None:
+    """
+    Create a kanaal, if it doesn't exist yet.
+    """
+    client = NotificationsConfig.get_client()
+
+    # look up the exchange in the registry
+    _kanaal = next(k for k in KANAAL_REGISTRY if k.label == kanaal)
+
+    kanalen = client.list("kanaal", query_params={"naam": kanaal})
+    if kanalen:
+        raise KanaalExists()
+
+    # build up own documentation URL
+    domain = Site.objects.get_current().domain
+    protocol = "https" if settings.IS_HTTPS else "http"
```

Comment:
Should document the need for this custom setting - it's not a standard Django setting.